### ISSUE TITLE
`nimbledeps` folder is ignored

### ISSUE THREAD

**Issue by markspanbroek** [asdf-community/asdf-nim#7]
**Describe the bug**

The `nimbledeps` folder is ignored when invoking nimble. This is a feature of nimble that is used for [project local dependencies](https://github.com/nim-lang/nimble#nimbles-folder-structure-and-packages).

**Steps to reproduce**

1. Create .nimble file with a dependency, for instance:

```
version = "0.1.0"
author = "bug reporter"
description = "showcases a bug in asdf-nim"
license = "MIT"

requires "asynctest"
```

2. Create an empty `nimbledeps` folder next to the .nimble file.
3. Invoke `nimble install -d -y`
4. Look inside the `nimbledeps` folder, and see that it's empty.

**Expected behavior**

At step 4 the `nimbledeps/pkgs` folder should contain the downloaded dependencies.

**Screenshots**

```
user@work:~/tmp$ cat reproduce.nimble 
version = "0.1.0"
author = "bug reporter"
description = "showcases a bug in asdf-nim"
license = "MIT"

requires "asynctest"
user@work:~/tmp$ mkdir nimbledeps
user@work:~/tmp$ nimble install -d -y
   Warning: Using the environment variable: NIMBLE_DIR='/home/user/.asdf/installs/nim/1.4.8/nimble'
  Verifying dependencies for reproduce@0.1.0
 Installing asynctest@any version
Downloading https://github.com/markspanbroek/asynctest using git
  Verifying dependencies for asynctest@0.2.1
 Installing asynctest@0.2.1
   Success: asynctest installed successfully.
  asdf-nim: regenerated shims

user@work:~/tmp$ ls nimbledeps/
user@work:~/tmp$ 
```

**Additional context**

It seems that asdf-vm sets the `NIMBLE_DIR` environment variable, which overrides the default behaviour of nimble when a nimbledeps folder is present.

**Comment by elijahr** [comment]
@markspanbroek thank you for the report!

I think this is fixed now on main (and in v1.1.5), via https://github.com/asdf-community/asdf-nim/commit/fdc9e4863403564f3412bd491febe454867af59f#diff-bd7e16b2f036811f6a156b2f59efc2e7621107b5fc9240c020fdd8caf127095eR16.

When you get a chance, can you confirm and let me know?

**Comment by markspanbroek** [comment]
Works like a charm. Thanks @elijahr!
